superset/utils.py

```diff
@@ -841,3 +841,45 @@ def ensure_path_exists(path):
     except OSError as exc:
         if not (os.path.isdir(path) and exc.errno == errno.EEXIST):
             raise
+
+
+def split_adhoc_filters_into_base_filters(fd):
```

Comment:
Typically we use the `form_data` variable to represent form data. It's probably best to rename `fd` to `form_data` for consistency.

Reply:
@john-bodley `fd` is used throughout the codebase already